qiskit/transpiler/passes/scheduling/alignments/reschedule.py

```diff
@@ -213,27 +222,17 @@ def run(self, dag: DAGCircuit):
         node_start_time = self.property_set["node_start_time"]
 
         for node in dag.topological_op_nodes():
-            if node_start_time[node] == 0:
-                # Every instruction can start at t=0
-                continue
-
-            if isinstance(node.op, Gate):
-                alignment = self.pulse_align
-            elif isinstance(node.op, Measure):
-                alignment = self.acquire_align
-            else:
-                # Directive or delay. These can start at arbitrary time.
-                continue
 
             try:
-                misalignment = node_start_time[node] % alignment
-                if misalignment == 0:
-                    continue
-                shift = max(0, alignment - misalignment)
+                start_time = node_start_time[node]
             except KeyError as ex:
                 raise TranspilerError(
                     f"Start time of {repr(node)} is not found. This node is likely added after "
                     "this circuit is scheduled. Run scheduler again."
                 ) from ex
```

Comment:
I think this pattern is much cleaner.
```
start_time = node_start_time.get(node)
if start_time is None:
        raise TranspilerError(....
```

Reply:
Agreed, done in [6d21886](https://github.com/Qiskit/qiskit-terra/pull/10077/commits/6d21886a1c6f0786bcba8c73e94e505cc19f0ef7).